proto/engine/v1/json_marshal_unmarshal.go

```diff
@@ -325,6 +327,14 @@ type ExecutionPayloadElectraJSON struct {
 	DepositRequests    []DepositRequestV1    `json:"depositRequests"`
 }
 
+// ExecutionPayloadBody represents the engine API ExecutionPayloadV1 or ExecutionPayloadV2 type.
+type ExecutionPayloadBody struct {
+	Transactions       []hexutil.Bytes       `json:"transactions"`
+	Withdrawals        []*Withdrawal         `json:"withdrawals"`
```

Comment:
why do Withdrawals need to be a pointer but WithdrawalRequests not a pointer?

Reply:
`(Deposit|Withdrawal)Requests` are a new type where I'm trying out something different, whereas `Withdrawals` is uses the existing pattern with its own un/marshal methods. I want to compare the ux of the two approaches - see also the functional helpers that the new types use.